samanthas_telegram_bot/conversation/auxil/message_sender.py

```diff
@@ -104,58 +114,72 @@ async def ask_store_username(update: Update, context: CUSTOM_CONTEXT_TYPES) -> N
     def _prepare_message_for_review(update: Update, context: CUSTOM_CONTEXT_TYPES) -> str:
         """Prepares text message with user info for review, depending on role and other factors."""
         data = context.user_data
-        locale = data.locale
+        locale: Locale = data.locale
 
         message = (
```

Comment:
I don't like the way messages are built here. At least, it's better to put context.bot_data.phrases to variable, to make code shorter. I think as above with `data`, to make function smaller and more readable, this `message`-building should be put to external functions

Reply:
Done (extracted into a variable). Other than that, I'm not sure what I can do to make it shorter. And this function is already extracted (i.e. it only builds a message, doesn't send it). Ofc, I can move it to another module, but not sure what benefit it will bring.

Reply:
This function is just too big imho. I was thinking about splitting it somehow, but don't have any idea. So let's keep it as it is then